README.md

```diff
@@ -2,10 +2,17 @@
 
 [![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/performance/performance-ci?branchName=master)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=271&branchName=master)
 
-This repo contains benchmarks used for testing the performance of .NET Frameworks.
+This repo contains benchmarks used for testing the performance of all .NET Runtimes: .NET Core, Full .NET Framework, Mono and CoreRT.
 
-See the [Microbenchmarks Guide](./src/benchmarks/micro/README.md) for information on running our microbenchmarks.
-See the [Real-World Scenarios Guide](./src/benchmarks/real-world/JitBench/README.md) for information on running our real-world scenario benchmarks.
+We know that having the benchmarks in a separate repository might seem to be surprising. But the performance is vertical. .NET Runtime features affect base class library performance. Which at the end, affects the end user and provides the business value. This is why we keep all the benchmarks in a single place - to test the performance of the end product. For whatever .NET Runtime you want.
```

Comment:
nit: consdier rewriting to to have fewer sentence fragements. Maybe:
"Finding these benchmarks in a separate repository might be surprising. Performance in a given scenario may be impacted by changes in seemingly unrelated components. Using this central repository ensures that measurements are made in comparable ways acrosss all .NET runtimes and repos. This consistency lets engineers make progress and ensures the customer scenarios are protected."

Reply:
thanks for providing the suggestion, I am going to change it